app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,7 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @posts = @user.posts.all
+    @posts = @user.posts
```

Comment:
- [ ] Kindly resolve the n+1 query for posts by chaining the comments with it. see [reference](https://api.rubyonrails.org/v6.1.4.4/classes/ActiveRecord/QueryMethods.html#method-i-includes)

Reply:
resolved. thank you for your review.